### ISSUE TITLE
Download notification bug

### ISSUE THREAD

**Issue by MaSHiNiK** [KotatsuApp/Kotatsu#272]
### Brief summary

After completing the download of any manga, the download notification hangs and is removed only after closing the application in the settings.

### Steps to reproduce

![IMG_20221212_175629_620](https://user-images.githubusercontent.com/80310257/207093634-3487571b-d1a0-4b29-8fe0-e6e4444ccf9f.jpg)

### Kotatsu version

4.0.6

### Android version

8.0

### Device

Huawei MediaPad T5 10

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
